src/Lib.hs

```diff
@@ -10,6 +10,8 @@ import Lib.App (AppEnv (..))
 import Lib.Core.Jwt (mkRandomString)
 import Lib.Server (API, server)
 
+import System.Remote.Monitoring (forkServerWith)
```

Comment:
This import should be moved to hackage dependencies section. Neat `Servant.Server` in this module.